openstack/nova/templates/scheduler-deployment.yaml

```diff
@@ -35,8 +35,8 @@ spec:
         configmap-etc-hash: {{ include (print $.Template.BasePath "/etc-configmap.yaml") . | sha256sum }}
         configmap-scheduler-etc-hash: {{ include (print $.Template.BasePath "/scheduler-etc-configmap.yaml") . | sha256sum }}
     spec:
-{{ tuple . "nova" "scheduler" | include "kubernetes_pod_anti_affinity" | indent 6 }}
-{{ include "utils.proxysql.pod_settings" . | indent 6 }}
+      {{- tuple . "nova" "scheduler" | include "kubernetes_pod_anti_affinity" | nindent 6 }}
+      {{- include "utils.proxysql.pod_settings" . | nindent 6 }}
```

Comment:
Why does `nova-scheduler` not need `utils.topology.constraints`?

Reply:
We only run one instance of it, and if we run more than one instance, we might get into trouble. AFAIK.

Reply:
hm ... correct, we currently do, but we also define `kubernetes_pod_anti_affinity` anyways, so I would have expected differently. anyways, no need to change currently, I guess. thanks.